### DISCUSSION TITLE
Unified detailed list of contents/index for articles

### DISCUSSION THREAD

**Discussion by D4n2021** [mdn/mdn-community#399] [Platform] [answered]
For example in this [Content scripts article ](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts), the 6 subheadings are completely missing in the list of contents on the left side:

[Loading content scripts](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts#loading_content_scripts)
[Restricted domains](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts#restricted_domains)
[Content script environment](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts#content_script_environment)
[Communicating with background scripts](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts#communicating_with_background_scripts)
[Communicating with the web page](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts#communicating_with_the_web_page)
[Using eval() in content scripts](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts#using_eval_in_content_scripts)

Not to mention even the 3-4 sub-subheadings of each of those 6 subheadings.

Please add those subheadings to the index ASAP (by using some script, I guess, that could be the fastest way), instead of having them on the right side, where barely any user would think of taking a look.

![grafik](https://github.com/mdn/mdn-community/assets/20766261/17c137bb-2224-4520-92bf-ff7a17cbb2de)

**Comment by yin1999** [comment, marked as answer]
Hi @D4n2021. The left sidebar is the place to hold related documents (not the TOC). You could find the TOC at the right side :)

![image](https://github.com/mdn/mdn-community/assets/15844309/bae0cc5c-f7b4-416c-8daa-713d69f716c7)
